SPECS/qt5-qtbase/qt5-qtbase.spec

```diff
@@ -256,6 +258,7 @@ Qt5 libraries used for drawing widgets and OpenGL items.
 %if 0%{?fedora} > 30
 %patch80 -p1 -b .use-wayland-on-gnome.patch
 %endif
+%patch81 -p1
```

Comment:
Optional suggestion: There is an `%autopatch` macro that we could use to clean up the individual application of patches here. You could do something like `%autopatch -p1 4 8 50 51 52 53 54 61 64 68 81` instead of all of the %patchXX macros that might be cleaner if you wanted to try it. 

documentation if you are curious: https://github.com/rpm-software-management/rpm/blob/master/docs/manual/autosetup.md